CIP-0105/README.md

```diff
@@ -237,6 +237,134 @@ This is because we have modeled DRep IDs and CC credentials on stake pool operat
 
 The advantage of including a network tag would be to reduce the likelihood of mislabelling a DRep’s network of operation (eg Preview v Cardano mainnet).
 
+### Test vectors
+
+Scripts were constructed according to the following script templates:
+
+`script_1: all [keyhash, active_from 5001]`
+`script_2: any [keyhash, all [active_from 5001, active_until 6001]]`
+
+where
+
+`keyhash: {drep | cc_cold | cc_hot}`
+
+#### Test vector 1
+`mnemonic: test walk nut penalty hip pave soap entry language right filter choice`
+`accountIx: 0x80000000`
```

Comment:
```suggestion
`accountIx: 0x00000000`
```
I believe this is a mistake.
The following key provided were produced using index of `0x00000000`.

Reply:
whoops, I was hardening the account index twice, disregard this